definitions/tools/bam_to_bigwig.cwl

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'cgpBigWig Converting BAM to BigWig'
+baseCommand: ["bam2bw", "-F", "1024", "-o", "MarkedSorted.bam.bw"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "quay.io/biocontainers/cgpbigwig:1.4.0--h93d22ca_0"
+    - class: ResourceRequirement
+      ramMin: 32000
+
+inputs:
+    bam:
+        type: File
+        inputBinding:
+            position: 1
+            prefix: '-i'
+        secondaryFiles: [^.bai]
+    reference:
+        type:
+            - string
+            - File
+        inputBinding:
+            position: 2
+            prefix: '-r'
+        secondaryFiles: [.fai, ^.dict]
+
```

Comment:
While you're messing around with it, could you expose another option or two?
https://github.com/cancerit/cgpBigWig/blob/develop/cwls/bam2bw.cwl exposes some additional options that seem like they'd be worthwhile.  Specifically, the option to handle overlapping reads seems important (and should probably be enabled by default, I think). 

Also, what's up with the scaling factor input to this tool?
```
-S  --scale-log10 [float]                         A scale factor to multiply to output [default: -1912244352]
```
Why is that the default value, and is that appropriate for both human and mouse here?

Reply:
we've verified that despite confusing docs, the scaling factor is not an issue and shouldn't be exposed here